src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.Managed.cs

```diff
@@ -24,7 +24,11 @@ private bool ChangeViewNative(double? horizontalOffset, double? verticalOffset,
 			=> (_presenter as ScrollContentPresenter)?.Set(horizontalOffset, verticalOffset, disableAnimation: disableAnimation) ?? true;
 
 		private partial void OnLoadedPartial() { }
-		private partial void OnUnloadedPartial() { }
+
+		private partial void OnUnloadedPartial()
+		{
+			Presenter?.UnhookScrollEvents(this);
```

Comment:
We `UnHook` on un-load, but we are not re-subscribing on next load. Also as we already have a ref to the `Owner`, I think it would be better to just keep everything in the SCP and just subscribe to the load/unload events from it

Reply:
Is this still relevant ? (If so, dismiss my comment about making the method `private` ;) )